src/cursor/cur_version.c

```diff
@@ -721,12 +722,12 @@ __wt_curversion_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner
     WT_ERR(version_cursor->file_cursor->close(version_cursor->file_cursor));
     WT_ERR(__wt_open_cursor(session, uri, cursor, file_cursor_cfg, &version_cursor->file_cursor));
 
-    /* Open the history store cursor for operations on the regular history store .*/
-    if (F_ISSET(S2C(session), WT_CONN_HS_OPEN)) {
+    /* Open the history store cursor for btrees may have data in the history store.*/
```

Comment:
Should this be `... for btrees that may have ...`?

Reply:
fixed